app.rb

```diff
@@ -87,6 +87,27 @@ def fixture(path)
       end
     end
 
+    # USPS IPPaaS auth
+    post "/oauth/authenticate" do
+      sleep ENV['USPS_IPPAAS_AUTH_DELAY'].to_f
```

Comment:
Should we document the new environment variables in README.md as is done with the others?

Reply:
Sure, I added documentation for those.